gossip/src/crds.rs

```diff
@@ -320,6 +325,7 @@ impl Crds {
                     }
                     let num_push_dups = entry.num_push_recv.unwrap_or_default();
                     entry.num_push_recv = Some(num_push_dups.saturating_add(1));
+                    self.stats.lock().unwrap().num_duplicate_push += 1;
```

Comment:
There is no lock contention on `self.stats` (because of `&mut self`), but still repeated `self.stats.lock().unwrap()`s might be inefficient.
Better to just lock `self.stats` once in the beginning of this function.